packages/hardhat-ledger/README.md

```diff
@@ -0,0 +1,90 @@
+[![npm](https://img.shields.io/npm/v/@nomicfoundation/hardhat-ledger.svg)](https://www.npmjs.com/package/@nomicfoundation/hardhat-ledger) [![hardhat](https://hardhat.org/buidler-plugin-badge.svg?1)](https://hardhat.org)
+
+# hardhat-ledger
+
+[Hardhat](https://hardhat.org) plugin for integration with a [Ledger hardware wallet](https://www.ledger.com/).
+
+## What
+
+This plugin extends the Hardhat provider enabling it to work with a connected Ledger wallet seamlessly.
+
+## Installation
+
+```bash
+npm install --save-dev @nomicfoundation/hardhat-ledger
+```
+
+And add the following statement to your `hardhat.config.js`:
+
+```js
+require("@nomicfoundation/hardhat-ledger");
+```
+
+Or, if you are using TypeScript, add this to your `hardhat.config.ts`:
+
+```js
+import "@nomicfoundation/hardhat-ledger";
+```
+
+## Tasks
+
+This plugin creates no additional tasks.
+
+## Environment extensions
+
+This plugin adds nothing to the Hardhat Runtime Environment.
+
+### Provider extensions
+
+The provider supplied by Hardhat will be extended using [`extendProvider`](https://hardhat.org/hardhat-runner/docs/advanced/building-plugins#extending-the-hardhat-provider), decorating it to be a `LedgerProvider`. Any other calls to extendProvider will be added on top of this.
+
+A `LedgerProvider` knows how to connect and interact with a Ledger wallet
+
+## Usage
```

Comment:
I wanted to write an "advanced usage" section detailing how to extend the provider by hand, importing `LedgerProvider` but I'm not sure that's encouraged. Let me know if you think is a good idea.

Nevertheless I've exported the errors from `index.ts` just in case the use-case of catching a particular derivation/connection/etc error arises.